src/pages/tutorial/task/task.js

```diff
@@ -103,6 +104,15 @@ class TaskPage extends React.Component {
     setProgress(progress);
   };
 
+  areDocsAttributesReady = (attrs) => {
```

Comment:
I think we could make this walkthrough-independent by just ensuring the values of every attribute that's provided is not `null/undefined`?

Also, where did you get the gif from? It's awesome!

Reply:
I made it myself on loading.io :) I tried that initially, but some of the values are always null in some of the walkthroughs.

Reply:
@philbrookes That issue you're seeing should end up being resolved by https://github.com/integr8ly/tutorial-web-app/pull/103 , basically some variables are currently only populated on the landing page so it causes that issue.

We can leave it for now anyway.

I think these two PRs will end up quite similar in terms of styling and stuff. There might be a possibility of using the same component at some stage in the future. I'll create a card.